test/webrtc_test.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+using_task_library "comms_webrtc"
+import_types_from "iodrivers_base"
+import_types_from "base"
+
+describe OroGen.comms_webrtc.Task do
+    run_live
+
+    attr_reader :task1, :task2
+
```

Comment:
One other thing: you will have to start rustysignal in the `before` and stop it in `after`. You can't expect it to be already running during the tests.

Reply:
Do you remember where could I find an example to do that?